src/reducers/transactionReducer.ts

```diff
@@ -15,8 +15,8 @@
 // along with Parity Bridges UI.  If not, see <http://www.gnu.org/licenses/>.
 
 import type { InterfaceTypes } from '@polkadot/types/types';
+import moment from 'moment';
```

Comment:
You should not use momentJs (https://momentjs.com/docs/#/-project-status/)

Reply:
how sad ... moment.js was so useful even though that mutable object stuff which tended to be a headache.

Anyway this can be easily replaced by `Date.now()` how i think about it.